.travis.yml

```diff
@@ -5,12 +5,40 @@ language: python
 python:
   - 3.6
   - 3.7
+  - 3.8-dev
+
+env:
+  - INSTALL=sdist
+
+matrix:
+  include:
+  - python: 3.7
+    env: INSTALL=wheel
+  - python: 3.7
+    env: INSTALL=editable
+
 
 before_install:
-  - travis_retry python -m pip install --upgrade pip setuptools wheel
+  - travis_retry python -m pip install --upgrade pip setuptools wheel twine
+  - |
+    if [ "$INSTALL" = "sdist" ]; then
+        python setup.py sdist
+        twine check dist/niflow-manager*.tar.gz
+    elif [ "$INSTALL" = "wheel" ]; then
+        python setup.py bdist_wheel
+        twine check dist/niflow_manager*.whl
+    fi
 
 install:
-  - python -m pip install ".[test]"
+  - |
+    if [ "$INSTALL" = "sdist" ]; then
+        pip install "$( ls dist/niflow-manager*.tar.gz )[test]"
+    elif [ "$INSTALL" = "wheel" ]; then
+        pip install "$( ls dist/niflow_manager*.whl )[test]"
+    elif [ "$INSTALL" = "editable" ]; then
+        pip install -e ".[test]"
```

Comment:
Just a note that I added this test to make sure that weirdness doesn't arise from the fact that we have these fake Python files in the data directory.